src/ui/Views/Course/Index.cshtml

```diff
@@ -94,9 +94,9 @@
           </td>
           <td class="govuk-table__cell">
               @if(course.IsRunning()) {
-                  <a href="@searchAndCompareUrlService.GetCoursePageUri(course.Provider.ProviderCode, course.CourseCode)">Yes - view online</a>
+                <a href="@searchAndCompareUrlService.GetCoursePageUri(course.Provider.ProviderCode, course.CourseCode)">Yes - view online</a>
               } else {
-                <p class="govuk-body govuk-!-margin-0">@(course.IsNotRunning() ? "No" : "No – must be published on UCAS")</p>
+                <p class="govuk-body govuk-!-margin-0">@((course.IsNotRunning() && Model.ProviderOptedIn) ? "No - still in draft" : "No - must be published on UCAS")</p>
```

Comment:
This logic isn't right.
A not running course will always say "No"
A new and not yet running course for opted in providers: No – still in draft
A new and not yet running course for non-opted in providers: No – must be published on UCAS